src/components/PopupWithForm.js

```diff
@@ -6,6 +6,7 @@ export default class PopupWithForm extends Popup {
 
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._inputList = this._popupForm.querySelectorAll(".modal__input");
+    this.avatarPic = document.querySelector("#profile__avatar-pic");
```

Comment:
`    this.avatarPic = document.querySelector("#profile__avatar-pic");
`

Please, delete it from `PopupWithForm`

There should **not** be any code in universal functions that doesn’t suit every popup.